PowerEditor/src/ScintillaComponent/columnEditor.cpp

```diff
@@ -472,6 +473,23 @@ void ColumnEditorDlg::switchTo(bool toText)
 
 	::SetFocus(toText?hText:hNum);
 
+	if (toText) 
+	{
+		ColumnEditorParam colEditParam = NppParameters::getInstance()._columnEditParam;
+		if (!colEditParam._insertedTextContent.empty())
+		{
+			::SetDlgItemText(_hSelf, IDC_COL_TEXT_EDIT, colEditParam._insertedTextContent.c_str());
+		}
+		else
+		{
+			::EnableWindow(::GetDlgItem(_hSelf, IDOK), false);
+		}
+	}
+	else 
+	{
+		::EnableWindow(::GetDlgItem(_hSelf, IDOK), true);

```

Comment:
EnableWindow(::GetDlgItem(_hSelf, IDOK), xxx); 
should be call only once at the end - xxx is the result of IDC_COL_TEXT_EDIT (empty or not).